apps/platform-app/src/backend/profile/profile.service.ts

```diff
@@ -142,12 +144,35 @@ export const findGithubUsernameByProfileId = async (profileId: string) => {
   })
 
   if (!profile?.user.githubDetails?.username) {
-    throw new Error('User does not have a GitHub username')
+    return null
   }
 
   return profile.user.githubDetails.username
 }
 
+export const findLinkedinUsernameByProfileId = async (
+  profileId: Profile['id'],
+) => {
+  const profile = await prisma.profile.findFirst({
```

Comment:
In the case of searching on unique ID fields, prisma.findUnique works a little faster than prisma.findFirst.